docs/logging.md

```diff
@@ -0,0 +1,81 @@
+# Logging
+
+# Application-level logging in logit.io
+
+The application sends all Rails logs to [logit.io](https://logit.io/). This
+gives us a hosted ELK stack. ELK stands for Elasticsearch, Logstash, Kibana.
+It’s a open-source stack frequently used for log aggregation and visualisation.
+
+Kibana is the tool that you’ll use to explore the logs.
+
+To view the production logs in Kibana:
+
+1. Log in to logit.io.
+2. Click on the “DfE Claim” account.
+3. Find the “Production” stack and click “Launch Kibana”.
+
+## Missing logs in logit.io
+
+At the time of writing (2020-03-26), we have a problem where some log messages
+seem to be missing from Logit.io. We are trying to fix this in [this Trello
+card](https://trello.com/c/JN44De4l/1330-understand-why-expected-logs-arent-in-logit).
+
+# Azure logs
+
+We should aim for logit.io to be the single place developers need to look to
+find logs. However, it’s useful to know about some of the logs that are
+available from Azure.
+
+The `az` Azure command line tool has some useful commands for interacting with
+logs.
+
+## App Service
+
+The App Service handles HTTP traffic.
+
+There is some [Azure documentation](https://docs.microsoft.com/en-us/azure/app-service/troubleshoot-diagnostic-logs#access-log-files)
+about the logs available for an App Service. Here we go over some of
+the ways to extract these logs.
+
+### Docker logs
+
+To view Docker logs in production, you will need to elevate your privileges using a [PIM request](privileged-identity-management-requests.md).
+
+You can use the `az webapp log` commands to tail and download the logs.
+
+This will live tail the container logs and also the Docker host logs:
+```
+az webapp log tail --name s118p01-app-as --resource-group s118p01-app --subscription s118-teacherpaymentsservice-production
+```
+
+Or, you can download them with a browser:
+- current Docker logs: https://s118p01-app-as.scm.azurewebsites.net/api/logs/docker
+- previous Docker logs: https://s118d01-app-as.scm.azurewebsites.net/api/vfs/LogFiles/
+
+TODO what is current and previous?
```

Comment:
@robbiepaul do you know anything about these logs?

Reply:
I know that they don't tell us much, it's mainly output from Docker. The log files with `_default` in their names contain some stdout from inside the container. 

The current logs show instances that are currently in our App Service. The `LogFiles` one contains current and previous instances in the rotation.

Also you've listed the production one for the current Docker logs and development for the previous Docker logs

Reply:
Thanks, I'll update based on what you've said. Maybe these logs aren't very useful at the moment, but they will become more useful if we switch back to Rails logging to stdout.